docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '3.8'
+services:
+  python_app:
+    build: ./app
+    container_name: python_app
+    ports:
+      - 3000:3000
```

Comment:
as per first comment this is exposed on 8080 - add an issue to parameterize this

Reply:
and fix this!